src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/Period.scala

```diff
@@ -35,7 +35,8 @@ sealed class PeriodType(val sizeInMilliseconds: Long, format: String, truncateFi
     if (timeZone.isDefined) {
       sdf.setTimeZone(timeZone.get)
     }
-    sdf.format(date)
+    val formattedDate = sdf.format(date)
+    this.periodTypeName + "-" + formattedDate
```

Comment:
In some cases, I would agree wrt the readability of the formatted string, but I think this case seems more readable to me with simple concatenation (and saves the minor performance hit of the string interpolation).